_locales/pl/messages.json

```
@@ -0,0 +1,383 @@
+		"message": "Wyświetlaj najniższą zarejestrowaną cenę na stronach aplikacji"
+	},
+	"options_online_stats": {
+		"message": "Wyświetlaj bieżących graczy i maksymalne szczyty na stronach aplikacji"
+	},
+	"options_steamdb_last_update": {
+		"message": "Wyświetlaj ostatnią aktualizację składów na stronach aplikacji"
+	},
+	"options_button_app": {
+		"message": "Wyświetl przycisk na stronach aplikacji"
+	},
+	"options_button_pcgw": {
+		"message": "Wyświetl przycisk PCGamingWiki na stronach aplikacji"
+	},
+	"options_button_sub": {
+		"message": "Wyświetl przycisk na stronach paczek"
+	},
+	"options_button_video": {
+		"message": "Wyświetl przycisk na stronach wideo"
+	},
+	"options_button_gamehub": {
+		"message": "Wyświetl przycisk w hubach gier"
+	},
+	"options_button_gamecards": {
+		"message": "Wyświetl przyciski SteamDB i strony sklepu na stronie kart gier"
+	},
+	"options_link_subid": {
+		"message": "Wyświetl ID paczki dla każdej opcji zakupu"
+	},
+	"options_link_subid_widget": {
+		"message": "Wyświetl ID paczki w widżetach aplikacji"
+	},
+	"options_link_accountpage": {
+		"message": "Link do każdej paczki na stronie licencji"
+	},
+	"options_profile_calculator": {
+		"message": "Dodaj link do kalkulatora na profilach społecznościowych"
+	},
+	"options_enhancement_inventory_sidebar": {
+		"message": "Przesuń listę wyboru gier na stronie ekwipunku na bok"
+	},
+	"options_link_inventory": {
+		"message": "Połącz każdy prezent i kupon w swoim ekwipunku ze SteamDB"
+	},
+	"options_link_inventory_gift_subid": {
+		"message": "Prawidłowo określ ID paczki prezentu w twoim ekwipunku"
+	},
+	"options_enhancement_inventory_no_sell_reload": {
+		"message": "Nie odświeżaj ekwipunku podczas sprzedaży przedmiotu"
+	},
+	"options_enhancement_inventory_quick_sell": {
+		"message": "Dodaj przyciski szybkiej sprzedaży"
+	},
+	"options_enhancement_inventory_quick_sell_auto": {
+		"message": "Automatycznie akceptuj szybką sprzedaż"
+	},
+	"options_display_market_price_others": {
+		"message": "Wyświetl cenę rynkową przedmiotów w ekwipunkach innych użytkowników"
+	},
+	"options_enhancement_tradeoffer_url_items": {
+		"message": "Wspieraj parametry <code>for_item</code> i <code>my_item</code>"
+	},
+	"options_github_explanation": {
+		"message": "Zobacz wyjaśnienie na Githubie"
+	},
+	"options_enhancement_tradeoffer_no_gift_confirm": {
+		"message": "Zignoruj powiadomienia ofert wymiany \"ta transakcja wydaje się podejrzana\" i \"czy jesteś pewien, że jest to prezent?\""
+	},
+	"options_enhancement_tradeoffer_better_errors": {
+		"message": "Wyświetlaj lepsze komunikaty o błędach, zamiast tajemniczych numerów błędów Valve"
+	},
+	"options_enhancement_skip_agecheck": {
+		"message": "Pomiń stronę weryfikacji wieku"
+	},
+	"options_enhancement_skip_agecheck_description": {
+		"message": "Ustaw ciasteczka wieku i urodzin, aby zawsze były akceptowane"
+	},
+	"options_enhancement_no_linkfilter": {
+		"message": "Usuń wszystkie linki filtru linków"

```

```suggestion
		"message": "Usuń filtrowanie z odnośników"
```

Chodzi tutaj o konkretny ficzer pomijania linkfiltra, oryginalny string w EN też jest napisany trochę nie teges.